msu/systems/mod_settings/load.nut

```diff
@@ -92,15 +90,16 @@ local testSettingsSystem = function()
 			"NewCampaignOnly"
 		]
 	}
-}
+};
```

Comment:
No semi-colon at the end of function definitions

Reply:
That's not a function